client/src/app/api/log-info/route.ts

```diff
@@ -1,3 +1,18 @@
+// utils/gitUtils.js
+import { execSync } from 'child_process';
+
+export function getLastCommitInfo() {
+  try {
+    // Fetch the last commit SHA and message
+    const commitInfo = execSync('git log -1 --format=%H%n%B').toString().trim();
+    const [sha, message] = commitInfo.split('\n');
+    return { sha, message };
+  } catch (error) {
+    console.error('Failed to get last commit info:', error);
```

Comment:
Instead of throwing an error, I think that I would return an empty `sha` and empty `message` so that we can still access the environment variables even if we can't load the git information. What do you think?